packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -120,14 +125,21 @@ class OverlayEntry extends ChangeNotifier {
   /// Whether the [OverlayEntry] is currently mounted in the widget tree.
   ///
   /// The [OverlayEntry] notifies its listeners when this value changes.
-  bool get mounted => _mounted;
-  bool _mounted = false;
-  void _updateMounted(bool value) {
-    if (value == _mounted) {
-      return;
-    }
-    _mounted = value;
-    notifyListeners();
+  bool get mounted => _overlayStateMounted.value;
+
+  /// Whether the `_OverlayState`s built using this [OverlayEntry] is currently
+  /// mounted.
+  final ValueNotifier<bool> _overlayStateMounted = ValueNotifier<bool>(false);
```

Comment:
With this change, we don't need to extend ChangeNotifier anymore, right? There's nothing from it that this class uses, AFIKT. We could just implement the Listenable interface.

Reply:
Yeah only the docs but I think I overrode the docs for `dispose` anyways.